raiden_contracts/tests/fixtures/one_to_n.py

```diff
@@ -49,17 +49,17 @@ def f(
         sender: HexAddress,
         receiver: HexAddress,
         amount: int = 10,
-        expiration_block: int = None,
+        expiration_timestamp: int = None,
         chain_id: int = chain_id,
         one_to_n_address: HexAddress = one_to_n_contract.address,
     ) -> dict:
-        if expiration_block is None:
-            expiration_block = web3.eth.block_number + 10
+        if expiration_timestamp is None:
+            expiration_timestamp = web3.eth.get_block("latest").timestamp + 150  # type: ignore
```

Comment:
Would it make sense to create a type safe wrapper for `web3.eth.get_block("latest").timestamp` and use that everywhere?

Reply:
I definitely makes sense. I'll rework this part.